process/sync/baseSync.go

```diff
@@ -97,22 +98,25 @@ func (boot *baseBootstrap) loadBlocks(
 
 		for i := highestNonceInStorer - blockFinality - lastBlocksToSkip; i <= highestNonceInStorer; i++ {
 			if i > highestNonceInStorer-lastBlocksToSkip {
-				errNotCritical := boot.removeBlock(i, blockUnit, hdrNonceHashDataUnit)
-				if errNotCritical != nil {
-					log.Info(errNotCritical.Error())
-				}
+				err = boot.removeBlock(i, blockUnit, hdrNonceHashDataUnit)
```

Comment:
this error became not critical. Is it ok ?

Reply:
Changed back as it was a problem here anyway. The removing mechanism is good for cleaning the storer but if something goes wrong with it this should not be critical as anyway the node will request the missing info from network and than it will committed it back to the storer (overwrite if it is the same)